txWorker.go

```diff
@@ -19,8 +37,13 @@ func processTxs(unprocessedTxs <-chan string) {
 	// Control returns to txWorker() where the select{} repeats and the
 	// next transaction is grabbed.
 	defer catchAbortedTx()
+
 	cmd := parseCommand(<-unprocessedTxs)
-	go sendToAuditLog(cmd)
+
+	if !*noAudit {
+		loggableCmds <- cmd
+	}
```

Comment:
Yea it does, but I just spent 10m thinking about it and can't really see a better way.